api/commands/validate.ts

```diff
@@ -0,0 +1,50 @@
+import * as cronParser from "cron-parser";
+import { dataform } from "@dataform/protos";
+
+
+function validateEmail(email: string) {
+  const email_regex = /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i;
+  return email_regex.test(email);
+}
+
+export function validateSchedules(schedules: dataform.schedules.Schedules) {
```

Comment:
Just thinking ahead a bit - we are going to want to show this in the UI so we may want to instead build up a list of all the errors and return or throw that instead, so you can actually see multiple issues as the same time, otherwise you'll only see one error, fix it, then another, so on.

Reply:
+1, but please throw the final error (don't return it), the CLI *should* handle that nicely

Reply:
I don't have any command from the cli to execute this validation method

Reply:
I am returning the errors array, if there is no error it will return a empty array